land_claims.civmap.json

```diff
@@ -140,13 +151,18 @@
+{"name":"Yoahtl","shortname":"Yoahtl","color":"#ffcd30","declutter":1,"x":-4080,"z":6680,"id":"d6d85a26-db6c-482a-a640-f2fe044a5757","polygon":[[[-5808,6195],[-5832,6359],[-5716,6547],[-5704,7087],[-5748,7203],[-5688,7235],[-5504,7247],[-4945,7159],[-4761,7239],[-4509,7383],[-4413,7575],[-4329,7615],[-4133,7606],[-3885,7670],[-3809,7790],[-3757,7938],[-3565,8012],[-3557,7913],[-3458,7824],[-3445,7797],[-3415,7777],[-3321,7785],[-3234,7739],[-3223,7715],[-3203,7689],[-3168,7689],[-3137,7701],[-3124,7611],[-3132,7582],[-3165,7538],[-3157,7499],[-3116,7493],[-3091,7504],[-3073,7523],[-3037,7508],[-3008,7518],[-2986,7503],[-2933,7505],[-2919,7517],[-2882,7588],[-2851,7603],[-2816,7579],[-2725,7570],[-2675,7587],[-2611,7644],[-2586,7664],[-2565,7663],[-2522,7643],[-2318,7786],[-2317,7834],[-2401,7864],[-2374,7913],[-2268,7859],[-2222,7913],[-2220,7964],[-2177,8008],[-2126,7972],[-2027,8057],[-1933,7964],[-1884,8003],[-1831,8073],[-1818,8143],[-1771,8179],[-1720,8151],[-1682,8247],[-1597,8297],[-1574,8399],[-1535,8408],[-1483,8357],[-1461,8338],[-1413,8323],[-1398,8264],[-1160,8261],[-1115,8249],[-1009,8264],[-958,8200],[-911,8204],[-904,8138],[-863,8104],[-871,8051],[-846,8041],[-841,8004],[-1075,7827],[-1095,7685],[-1075,7623],[-965,7509],[-899,7571],[-757,7605],[-707,7533],[-559,7489],[-475,7404],[-417,7354],[-387,7252],[89,7252],[125,7234],[293,6935],[296,6892],[249,6830],[27,6892],[-163,7048],[-555,6997],[-746,7084],[-795,7161],[-1185,6919],[-1317,6945],[-1315,6881],[-1295,6849],[-1379,6739],[-1419,6729],[-1579,6787],[-1735,6643],[-1931,6725],[-1983,7009],[-2075,7071],[-2409,6815],[-2496,6695],[-2434,6567],[-2365,6393],[-2401,6284],[-2564,6184],[-2674,6206],[-2740,6310],[-2804,6254],[-2986,6188],[-3500,5698],[-3906,5712],[-4304,5604],[-4620,5812],[-4694,6041],[-4924,5960],[-5264,5906],[-5468,5968]]]},
+{"name":"Xi'an\n(Adina,Caledonia)","shortname":"Xi'an","color":"#15316a","declutter":3,"x":2770,"z":970,"id":"f1921958-9096-464f-a314-66addd14eda1","polygon":[[[2808,696],[2769,619],[2855,600],[2909,710],[2944,739],[2985,797],[3009,854],[3062,895],[3026,939],[2973,968],[2723,1235],[2661,1263],[2553,1154],[2553,1098],[2577,1069],[2577,992],[2569,990],[2569,974],[2562,974],[2561,960],[2568,959],[2568,952],[2621,886],[2706,850],[2743,813],[2776,734],[2808,696]]]},
```

Comment:
X before Y but I don't care, will get auto sorted next time I change this manually

Reply:
Oh damn it was the last one I added. got my abc's jumled up :/